Safewaan/HonkedIn
# C45choi/view resources

## Trello ticket link

[[Ticket Name](https://trello.com/b/RCwqUZD8/msci-342-scrum-board )](https://trello.com/c/5JpAZdfZ/120-as-a-user-i-want-to-be-able-to-see-all-resources-posted)



## Implementation description
* created the resources page where you can view all resources that have been submitted



## Steps to test
1. Ensure that routing to resources works
2. Ensure that you can see all the resources that were submitted into the DB
3. Ensure that when you press the link, it takes you to that website



## What should reviewers focus on?
*
